examples/apps/contact-list/package.json

```diff
@@ -14,17 +14,17 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "jitar": "^0.5.0",
+    "jitar": "^0.6.0",
```

Comment:
Was the upgrade of the Jitar version number required to make it work? If so, we need to also do this in our templates. If not, it's fine like this.

Reply:
I made the update to make the server start. It had installed 0.5.1 and it did not want to update to 0.6.0. The server kept failing as the startup script was provided as an array, but it still expected a string.

Reply:
I've updated all the package files to use jitar 0.6